src/commands/urbandictionary.js

```diff
@@ -35,9 +35,10 @@ module.exports = {
                 }
               })
             } else {
-              msg.channel.createMessage(`<@${msg.author.id}>, ${suffix}: This word is so screwed up, even Urban Dictionary doesn't have it in its database`)
+              msg.channel.createMessage(`<@${msg.author.id}>, ${suffix}: This word is so screwed up, even UrbanDictionary doesn't have it in its database`)
```

Comment:
Name mangling issue here. According to [Wikipedia](https://en.wikipedia.org/wiki/Urban_Dictionary), the name is in fact **Urban Dictionary**.

Reply:
Hah, accidentally changed this with my last commit, will fix.